### DISCUSSION TITLE
Auto connection for custom BSP

### DISCUSSION THREAD

**Discussion by name-snrl** [scalameta/metals#6256] [Q&A]
Hello, I'm using https://github.com/ValdemarGr/mezel as BSP, but after #6164 has been merged, it doesn't connect automatically. Is there a problem with connecting third-party BSPs? I don't really like the idea of connecting BSP manually.

**Comment by tgodzik** [comment]
Ideally it should work better, otherwise it looks like a bug. @jkciesluk any idea what might have happened?

**Reply by name-snrl** [reply]
> Ideally it should work

should I open an issue?

**Reply by jkciesluk** [reply]
Looks like a bug, maybe we no longer detect it as auto connectable. I will try to debug it with the example project from https://github.com/ValdemarGr/mezel/tree/main/example/src/main/scala/example

**Reply by jkciesluk** [reply]
> should I open an issue?

If you can, please.
